templates/extensions/extension.yaml

```diff
@@ -39,8 +39,6 @@ resources:
     description: >-
```

Comment:
Doesn't let me comment on the lines above for some reason... but on line 18 the sourceUrl is
`sourceUrl: https://github.com/firebase/firebase-tools/tree/master/templates/extensions`

Can we change it to something that's more obvious that it needs to be changed.
ex: `https://github.com/MY_ORG/MY_REPO/MY_EXTENSION`